src/components/App.tsx

```diff
@@ -40,6 +41,7 @@ function App() {
                   <Route path="classes/:classSlug" element={<ClassPage/>} />
                   <Route path="magicitems/:letterRange" element={<MagicItemsPage />} />
                   <Route path="monsters" element={<MonstersPage/>}/>
+                  <Route path="monsters/:monsterSlug" element={<IndvMonsterPage/>}/>
```

Comment:
Does Indv stand for Individual? I think I have used Single which can read better. But Indv is fine